src/commons/eventsourcing/store.rs

```diff
@@ -872,7 +881,7 @@ where
         }
 
         if aggregate_opt.is_none() {
-            debug!("No snapshots found for '{}' will try from initialization event.", id);
+            warn!("No suitable snapshot for '{}' will rebuild state from events.", id);
```

Comment:
What can or should an operator do with these warnings? Is warn the right level or should these new warnings be info level instead?

Reply:
I thought a warning might be appropriate because this can take some time. I could add "This can take some time".

Reply:
added